src/Microsoft.AspNetCore.SignalR.Client/HubConnection.cs

```diff
@@ -45,29 +46,29 @@ public class HubConnection
             remove { _connection.Closed -= value; }
         }
 
+        public HubConnection(Uri url, IInvocationAdapter adapter)
```

Comment:
How did we end up in overload hell so fast. Can we re-evaluate these constructors? Some are useful for testing, and some are useful for humans. What we have right now is not very well thought out.